src/rapids_dependency_file_generator/cli.py

```diff
@@ -51,35 +60,62 @@ def validate_args(argv):
         ],
     )
     codependent_args.add_argument(
+        "-m",
         "--matrix",
         help=(
             "String representing which matrix combination should be generated, "
             'such as `--matrix "cuda=11.5;arch=x86_64"`. May also be an empty string'
         ),
     )
 
+    parser.add_argument(
+        "--prepend-channel",
+        action="append",
+        default=[],
+        dest="prepend_channels",
+        help=(
+            "A string representing a conda channel to prepend to the list of "
+            "channels. This option is only valid with --output "
+            f"{OutputTypes.CONDA} or no --output. May be specified multiple times."
+        ),
+    )
     parser.add_argument(
         "--prepend-channels",
+        dest="prepend_channels_deprecated",
         help=(
             "A string representing a list of conda channels to prepend to the list of "
             "channels. Channels should be separated by a semicolon, such as "
             '`--prepend-channels "my_channel;my_other_channel"`. This option is '
-            f"only valid with --output {OutputTypes.CONDA} or no --output"
+            f"only valid with --output {OutputTypes.CONDA} or no --output. "
+            "DEPRECATED: Use --prepend-channel instead."
         ),
     )
 
     args = parser.parse_args(argv)
+
+    if args.file_key_deprecated:
```

Comment:
Don't let users specify both flags.
```suggestion
    if args.file_key_deprecated:
        if args.file_key:
            raise ValueError("The --file_key (deprecated) and --file-key arguments cannot be specified together.")
```